sample-kotlin-model/src/main/kotlin/com/vimeo/dummy/sample_kotlin/KotlinSamples.kt

```diff
@@ -0,0 +1,25 @@
+package com.vimeo.dummy.sample_kotlin
+
+import com.google.gson.annotations.SerializedName
+import com.vimeo.stag.UseStag
+
+/**
+ * Test kotlin model
+ *
+ * Created by restainoa on 5/8/17.
+ */
+@UseStag
+class KotlinSamples {
+    var stringField: String? = null
+
+    var nonNullStringField: String = "default"
+
+    var intField: Int? = null
+
+    var longField: Long = 1
+
+    @SerializedName("boolean_field")
+    var booleanField: Boolean? = null
+
+    var testField: Any? = null
```

Comment:
How does field visibility work in Kotlin? Is it fine to have these public? Should they be private with `internal` getters/setters?

Obviously since this is a sample it doesn't really matter - but I'm curious what type of visibility people usually use with Kotlin. Are setters exposed? I can imagine you'd want to make most classes externally immutable. Unless they're immutable by default?